MetaMask/metamask-extension
# Fix EIP-6963 Prod Build names

## **Description**

Fixes bug where all prod builds of MetaMask would announce their EIP-6963 provider as just `MetaMask` without build name. This PR updates the announced EIP-6963 provider name and keeps the manifest.json extension name as is.

## **Related issues**

Fixes: #

## **Manual testing steps**

1. `yarn build prod`
2. `yarn build prod --build-type mmi`
3. `yarn build prod --build-type flask`
4. Add each build to your browser
5. Visit https://eip6963.org/
6. Verify names

## **Screenshots/Recordings**



### **Before**



### **After**

![image](https://github.com/MetaMask/metamask-extension/assets/918701/1c68eb27-3ceb-4686-90fc-f73910a8eb99)
![Screenshot 2023-11-29 at 10 26 30 AM](https://github.com/MetaMask/metamask-extension/assets/918701/95c2a551-4eab-4fd8-9226-e3583aeb343f)


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
